### DISCUSSION TITLE
[DUA] MTD registers DUA and receives ADDR_ERR.ntf

### DISCUSSION THREAD

**Discussion by guerbasc** [openthread/openthread#7856] [Q&A] [answered]
5.9.13 test always failed with DUT as MTD {SED, SSED, MED} , DUT miss to  send Child update request , I think that DUT can't regenerated DUA dg2. 

Test description:
  The purpose of this test case is to verify that a MTD, that performs DUA registration 
  through its Parent Router, correctly handles an ADDR_ERR.ntf which indicates its DUA 
  is a duplicate.

Test Toplogy:
'C_5_9_13':{
		'BR_1':[DevCapb.C_BBR, 'Kirale'],
		'BR_1EXT':[DevCapb.C_BBR, 'OTNCS_BR'],
		'Router':[DevCapb.C_FFD, 'OTS12'],
		'RouterEXT':[DevCapb.C_FFD, 'OTS12'],
		'Host':[DevCapb.C_Host, 'any'],
		'MED':[DevCapb.C_RFD, 'OTNCS12'],

If there are ideas that can help me solve the problem, I'm interested!

Thanks for read me,

**Comment by jwhui** [comment, marked as answer]
This should be fixed by:
* #7844

**Reply by EricSTMicro** [reply]
Hello Jonathan, 

how is that possible that Kirale/Nordic/Silabs devices successfully pass this test (C_5_9_13) in GRL v55.0 (with FW from February/Mars), but this patch was added yesterday only?

Regards,
Eric

**Reply by EricSTMicro** [reply]
Ok I get it, the Kirale/Nordic/Silabs GUs run a FTD FW, it's only the MTD that is impacted by this patch.
